Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -0,0 +1,1325 @@
+    ioc_output = tableToMarkdown(f'The IOCs for the report', ioc_contents, removeNull=True)
+    results = CommandResults(
+        outputs_prefix='CarbonBlackEEDR.Report',
+        outputs_key_field='ID',
+        outputs=context,
+        readable_output=readable_output + ioc_output,
+        raw_response=result
+    )
+    return results
+
+
+def get_file_device_summary(client: Client, args: dict) -> CommandResults:
+
+    sha256 = args.get('sha256')
+    result = client.get_file_device_summary_request(sha256)
+    readable_output = tableToMarkdown('The file device summary', result)
+    results = CommandResults(
+        outputs_prefix='CarbonBlackEEDR.File',
+        outputs_key_field='sha256',
+        outputs=result,
+        readable_output=readable_output,
+        raw_response=result
+    )
+    return results
+
+
+def get_file_metadata_command(client: Client, args: dict) -> CommandResults:
+    sha256 = args.get('sha256')
+    result = client.get_file_metadata_request(sha256)
+    contents = {
+        'SHA256': result.get('sha256'),
+        'file_size': result.get('file_size'),
+        'internal_name': result.get('internal_name'),
+        'original_filename': result.get('original_filename'),
+        'comments': result.get('comments'),
+        'os_type': result.get('os_type')
+    }
+    context = {
+        'sha256': result.get('sha256'),
+        'architecture': result.get('architecture'),
+        'available_file_size': result.get('available_file_size'),
+        'charset_id': result.get('charset_id'),
+        'comments': result.get('comments'),
+        'company_name': result.get('company_name'),
+        'file_available': result.get('file_available'),
+        'file_description': result.get('file_description'),
+        'file_size': result.get('file_size'),
+        'file_version': result.get('file_version'),
+        'internal_name': result.get('internal_name'),
+        'lang_id': result.get('lang_id'),
+        'md5': result.get('md5'),
+        'original_filename': result.get('original_filename'),
+        'os_type': result.get('os_type'),
+        'product_description': result.get('product_description'),
+        'product_name': result.get('product_name'),
+        'product_version': result.get('product_version')
+    }
+
+    file = Common.File(
+        md5=result.get('md5'),
+        sha256=sha256,
+        size=result.get('file_size'),
+        company=result.get('company_name'),
+        product_name=result.get('product_name')
+    )
+
+    readable_output = tableToMarkdown('The file metadata', contents)
+    results = CommandResults(
+        outputs_prefix='CarbonBlackEEDR.File',
+        outputs_key_field='sha256',
+        outputs=context,
+        readable_output=readable_output,
+        raw_response=result,
+        indicators=[file]
+    )
+    return results
+
+
+def get_file_command(client: Client, args: dict) -> CommandResults:
```

Comment:
why are we not outputting to File context standard here?

Reply:
@Itay4 It returns just the sha256 and download link. Don't think it's necessary.
What do you want to do?